Python/sysmodule.c

```diff
@@ -955,6 +955,21 @@ sys_intern_impl(PyObject *module, PyObject *s)
     }
 }
 
+/*[clinic input]
```

Comment:
Can you move the function after _is_interned(), to keep intern() + _is_interned() together?